js/apps/account-ui/pom.xml

```diff
@@ -49,6 +49,31 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>jboss-release</id>
```

Comment:
Is this the correct profile, or should it be `distribution`? Not actually sure what `jboss-release` is used for.